app/server.R

```diff
@@ -203,14 +202,15 @@ server <- function(input, output, session) {
       input$asOf != "" &&
       nrow(PREV_AS_OF_DATA()) != 0 &&
       input$asOf < CURRENT_WEEK_END_DATE()) {
-      asOfData <- PREV_AS_OF_DATA() %>%
-        rename(target_end_date = time_value, as_of_actual = value) %>%
-        select(target_end_date, geo_value, as_of_actual)
+      asOfData <- PREV_AS_OF_DATA()
+      asOfData$target_end_date <- asOfData$time_value
+      asOfData$as_of_actual <- asOfData$value
+      asOfData <- asOfData[, c("target_end_date", "geo_value", "as_of_actual")]
```

Comment:
quibble: i'm lukewarm on this refactor

but if it's more performant or you feel strongly about it i could be convinced

Reply:
Yeah, it's ugly. The new version is 110x faster :exploding_head: but the absolute improvement in runtime is practically insignificant compared to other slow parts of the dashboard. I added it because it was quick. If you feel strongly that the old version is more readable, I'm on board.

Reply:
Maybe keep it, but add the dplyr equivalent as an explanatory comment?